index.html

```diff
@@ -0,0 +1,159 @@
+                        </div>
+                        <div class="card-content">
+                            <p>Let's start todays session with some warmup stretches!</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col s6 m4">
+                <div class="card">
+                    <div class="card-image">
+                        <img src="Assets/Images/CircuitTraining.jpg">
+                        <span class="card-title">Warmup</span>
+                        <a class="btn-floating halfway-fab waves-effect waves-light red"><i
+                                class="material-icons">video_call</i></a>
+                    </div>
+                    <div class="card-content">
+                        <p>Let's start todays session with some warmup stretches!</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col s6 m4">
+                <div class="card">
+                    <div class="card-image">
+                        <img src="Assets/Images/CircuitTraining.jpg">
+                        <span class="card-title">Warmup</span>
+                        <a class="btn-floating halfway-fab waves-effect waves-light red"><i
+                                class="material-icons">video_call</i></a>
+                    </div>
+                    <div class="card-content">
+                        <p>Let's start todays session with some warmup stretches!</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col s6 m4">
+                <div class="card">
+                    <div class="card-image">
+                        <img src="Assets/Images/CircuitTraining.jpg">
+                        <span class="card-title">Warmup</span>
+                        <a class="btn-floating halfway-fab waves-effect waves-light red"><i
+                                class="material-icons">video_call</i></a>
+                    </div>
+                    <div class="card-content">
+                        <p>Let's start todays session with some warmup stretches!</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    </div>
+    <br><br>
+    </div>
+
+    <footer class="page-footer orange">
+        <div class="container">
+            <div class="row">
+                <div class="col l6 s12">
+                    <!-- donation button goes here -->
+                    <a href="https://act.autismspeaks.org/site/Donation2?idb=135287805&4141.donation=form1&DONATION_LEVEL_ID_SELECTED=1&df_id=4141&mfc_pref=T&idb=[[S76:idb]]"
+                        button class="btn waves-effect waves-light" type="submit" name="action">Donate to Autism Speaks
+                        <i class="material-icons right">monetization_on</i>
+                        </button></a>
```

Comment:
Is the donation button color intentionally greyed out? Or do we need a little more contrast?

Reply:
we shall style the button at a later time :)